src/utils/Calendar.js

```diff
@@ -0,0 +1,23 @@
+export const getNewDateStateChange = ({ code, focusedDay, startDate, endDate }) => {
```

Comment:
I think this would also be a good place for jsdoc comments to indicate it returns an object that contains one of or both of currentDate and focusedDate as unix moment values or null.